typescript-test-samples/launchdarkly-lambda-dynamodb/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Typescript: LaunchDarkly, AWS Lambda and DynamoDB Example",
+  "description": "This project contains examples of integrating LaunchDarkly feature flags into Lambda.",
+  "content_language": "English",
+  "language": "TypeScript",
+  "type": ["Integration"],
+  "diagram": "/img/LaunchDarkly-AWS-diagram.png",
+  "framework": "CDK",
+  "services": ["lambda", "dynamodb"],
+  "git_repo_url": "https://github.com/aws-samples/serverless-test-samples",
+  "pattern_source": "AWS",
+  "pattern_detail_tabs": [
+    {
+      "title": "Lambda using LaunchDarkly",
+      "filepath": "/cdk/resources/lambdas/getFlagsStarter/index.mjs"
+    },
+    {
+      "title": "Lambda using LaunchDarkl with DynamoDB feature store",
+      "filepath": "/cdk/resources/lambdas/getFlagsWithDynamo/index.mjs"
+    },
+    {
+      "title": "Lambda for Syncing LaunchDarkly Flags to DynamoDB",
+      "filepath": "/cdk/resources/lambdas/getFlagsStarter/index.mjs"
```

Comment:
Incorrect file path (I think?)

Reply:
They appear to be correct to me. The `.ts` files are also in there so that they can regenerate the files if they need to from the TypeScript source but you can't deploy those directly without compiling them.

Reply:
Just the third one... isn't that supposed to be the `syncFlagsToDynamo` function rather than `getFlagsStarter`?

Reply:
Ah, yes. Sorry about that.